dev/local-environment/configurations/genesis/shelley/genesis.json

```diff
@@ -16,9 +16,9 @@
     }
   },
   "initialFunds": {
-    "008a75c2002312f380c7ceaf3b18d5ee5aa8a3d309ef40c80fca9006b2d9267776d1289412d8d3c09599393044730deba14feb31f52b0e6ef2": 3480000000000,
-    "00207e498c990230dd7db797eaa2de8b8c9fce9b522b5a0f7d658c2974d27cb9a1a408b81239b55ec03f6e004deffcc094e7252d5c45743784": 500000000,
-    "005281ccb1fceb0a3b038602500080dd771cccd6800dc753c9786ff5d3f441c3ef7ef4a8ade039f4f4224b9ae494125bbcff284df64e8e73d8": 500000000,
+    "008a75c2002312f380c7ceaf3b18d5ee5aa8a3d309ef40c80fca9006b2d9267776d1289412d8d3c09599393044730deba14feb31f52b0e6ef2": 5000000000,
```

Comment:
It is 'dave' stake. When it is hundreds times higher than 'eve', we won't see both producing blocks.